### ISSUE TITLE
Dynamax Moves not supported?

### ISSUE THREAD

**Issue by Corus86** [pmariglia/showdown#86]
Was using this and noticed that after dynamxing, the program still thinks that the moves are the normal variants and uses them instead of the dynamaxed move for its decision.

**Comment by Corus86** [comment]
I also noticed that the bot saves dynamax for the very last pokemon always without factoring this into it's calculation. Would it be possible for the bot to choose the right pokemon to dynamax to minimize loss?

**Comment by pmariglia** [comment]
Discussed a bit in #23

Yeah both are intentional. Dynamax as a mechanic is really hard to implement using a search algorithm that can only see 2 or 3 turns ahead. Since it was banned in major formats I decided to ignore it completely.

The compromise I added was that the bot will always dynamax on the last pokemon if dynamax is available, but won't understand the effects of the MaxMoves. I guess understanding of the MaxMoves could be added, but its really just not worth the effort in my opinion. I would accept pull-requests adding this feature though.
